src/SonataAdminBundle.php

```diff
@@ -35,5 +52,61 @@ public function build(ContainerBuilder $container): void
         $container->addCompilerPass(new GlobalVariablesCompilerPass());
         $container->addCompilerPass(new ModelManagerCompilerPass());
         $container->addCompilerPass(new ObjectAclManipulatorCompilerPass());
+        $container->addCompilerPass(new WebpackEntriesCompilerPass());
+
+        $this->registerFormMapping();
+    }
+
+    public function boot(): void
+    {
+        $this->registerFormMapping();
+    }
+
+    /**
+     * Register form mapping information.
+     */
+    public function registerFormMapping(): void
```

Comment:
All of this has been removed in `master`

Reply:
Thank you.